### ISSUE TITLE
Bug: Project's counter title is not working in Chinese

### ISSUE THREAD

**Issue by RobertIndie** [taigaio/taiga-front#1569]
When changing my language to Chinese and go to home page, it raises some errors in the console.And the counter title is not working.
![image](https://user-images.githubusercontent.com/16974619/45085238-cff19f80-b132-11e8-9acf-4a95bef73548.png)
![image](https://user-images.githubusercontent.com/16974619/45085254-d8e27100-b132-11e8-81e2-6325a0211305.png)

**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

Raise some errors. The counter title is not working.

**If the current behavior is a bug, please provide the steps to reproduce.**

Change the language to Chinese and refresh home page. It will raise some errors in console.
Hover mouse on the home page -> project list -> project -> fans counter or watchers counter or members counter, it will show some incorrect message such as "{{ 'PROJECT.MEMBERS_COUNTER_TITLE'|translate:{total:project.get('members').size||0}:'messageformat' }}"

**What is the expected behavior?**

Show the message like "0 fans".

**Is it happening in taiga.io or in your own instance?**

Both.

**What browser/version are you using?**

Chrome and Firefox

**Are there any console errors *(Ctrl + F12)* in red?**

```
Error: [$interpolate:interr] Can't interpolate: {{ 'PROJECT.FANS_COUNTER_TITLE'|translate:{total:project.get('total_fans')||0}:'messageformat' }}
Error: Invalid key `one` for argument `total`. Valid plural keys for this locale are `other`, and explicit keys like `=0`.
http://errors.angularjs.org/1.5.5/$interpolate/interr?p0=%7B%7B%20'PROJECT.FANS_COUNTER_TITLE'%7Ctranslate%3A%7Btotal%3Aproject.get('total_fans')%7C%7C0%7D%3A'messageformat'%20%7D%7D&p1=Error%3A%20Invalid%20key%20%60one%60%20for%20argument%20%60total%60.%20Valid%20plural%20keys%20for%20this%20locale%20are%20%60other%60%2C%20and%20explicit%20keys%20like%20%60%3D0%60.
    at angular.js:68
    at Function.no.interr (angular.js:11829)
    at l.exp (angular.js:12134)
    at pre (angular.js:9585)
    at dt (angular.js:9694)
    at d (angular.js:9072)
    at a (angular.js:8397)
    at a (angular.js:8400)
    at d (angular.js:9088)
    at a (angular.js:8397)
    at d (angular.js:9088)
    at a (angular.js:8397)
    at angular.js:8277
    at angular.js:8615
    at r (angular.js:8414)
    at f (angular.js:9143)
    at tg-repeat.js:301
    at p.$digest (angular.js:17073)
    at p.$apply (angular.js:17337)
    at angular.js:17633
    at o (angular.js:5869)
    at angular.js:6145
    at r (raven.js:445)
```
```
Error: Invalid key `one` for argument `total`. Valid plural keys for this locale are `other`, and explicit keys like `=0`.
    at messageformat.js:2685
    at Array.forEach (<anonymous>)
    at st (messageformat.js:2684)
    at m (messageformat.js:3228)
    at h (messageformat.js:2920)
    at f (messageformat.js:2904)
    at o (messageformat.js:4323)
    at r.compile (messageformat.js:158)
    at i.compile (messageformat.js:639)
    at Object.i.interpolate (angular-translate-interpolation-messageformat.js:141)
    at rt (angular-translate.js:1862)
    at Function.B.instant (angular-translate.js:2343)
    at n (angular-translate.js:3223)
    at fn (eval at compile (angular.js:14432), <anonymous>:4:482)
    at angular.js:15525
    at p.$digest (angular.js:17064)
    at p.$apply (angular.js:17337)
    at angular.js:17633
    at o (angular.js:5869)
    at angular.js:6145
    at r (raven.js:445)
```

**Comment by alexhermida** [comment]
seems to be an error with the Chinese translation strings and pluralization. Thanks for reporting.

**Comment by yungang** [comment]
in dist/v-1539613280826/locales/taiga/locale-zh-hans.json file
delete all json elements that contains "one", like:
"COUNTER_TITLE":"{total, plural, one{one watcher} other{# watchers}}"

**Comment by coodeneko** [comment]
Also having this error in Japanese. 

Was previously reported in the [Google Group Forum](https://groups.google.com/forum/#!topic/taigaio/Y_tiXrJ92NE) by another user but also experiencing the issue myself on a self-hosted instance.

**Comment by rushfly** [comment]
I have same issue but resolved.
In taiga front locale file like a "locale-zh-hans.json", change the text "one" to "1", such as
 1{一个关注者}

**Comment by migonzalvar** [comment]
Finally solved in all languages through #1841
